src/__tests__/augmentConfig-test.js

```diff
@@ -0,0 +1,32 @@
+import standalone from "../standalone"
+import test from "tape"
+
+test("Setting `processors` inside `config` object should load the processors", t => {
+  standalone({
+    code: "a { b: \"c\" }",
+    configBasedir: __dirname,
+    config: {
+      processors: ["./fixtures/processor-invalid-transformation"],
+      rules: [],
+    },
+  }).then(({ results }) => {
+    t.equal(results[0].errored, true)
+    t.end()
+  }).catch(t.end)
+})
+
+test("Setting `processors` inside `configOverrides` object should override the one set in `config` object", t => {
+  standalone({
+    code: "a { b: \"c\" }",
+    configBasedir: __dirname,
+    config: {
+      rules: [],
+    },
+    configOverrides: {
+      processors: ["./fixtures/processor-invalid-transformation"],
+    },
+  }).then(({ results }) => {
+    t.equal(results[0].errored, true)
+    t.end()
+  }).catch(t.end)
+})
```

Comment:
Since these tests are not testing `augmentConfig` specifically, but actually test `standalone`, let's put them in one of the other testfiles named to address `standalone`.

I'm not sure the assertion `t.equal(results[0].errored, true` is specific enough. There could be an error for other reasons. I suggest using one of the plugins that already exists & is involved in tests, adding it via `configOverrides`, then asserting that it caused warnings that would not have been caused if the plugin was not loaded.